spicedb/docker-compose.yaml

```diff
@@ -0,0 +1,62 @@
+---
+# This runs SpiceDB, using Postgres as the storage engine. SpiceDB will not have
+# any schema written to it.
+#
+# Once the database service is running, the migrate service executes, running
+# "spicedb migrate head" to migrate Postgres to the most current revision. After
+# Postgres is migrated, the migrate service will stop.
+#
+# SpiceDB settings:
+#   pre-shared key: foobar
+#   dashboard address: http://localhost:8080
+#   metrics address: http://localhost:9090
+#   grpc address: http://localhost:50051
+#
+# Postgres settings:
+#   user: postgres
+#   password: secret
+#   port: 5432
+
+version: "3"
+
+services:
+  rebac:
+    image: "quay.io/ciam_authz/insights-rebac:latest"
+    profiles: ["rebac"]
+    restart: "always"
+    ports:
+      - "8000:8000"
+      - "9000:9000"
+
+  spicedb:
+    image: "authzed/spicedb"
+    command: "serve"
+    restart: "always"
+    ports:
+      - "8080:8080"
+      - "9090:9090"
+      - "50051:50051"
+    environment:
+      - "SPICEDB_GRPC_PRESHARED_KEY=${SPICEDB_GRPC_PRESHARED_KEY}"
```

Comment:
Do we want to provide a bootstrap yaml?
e.g. SPICEDB_DATASTORE_BOOTSTRAP_FILES=/mnt/spicedb_bootstrap.yaml
(see https://github.com/merlante/inventory_access_poc/blob/main/docker-compose.yml)